src/pages/workspace/accounting/qbo/import/QuickbooksLocationsPage.tsx

```diff
@@ -21,6 +21,7 @@ function QuickbooksLocationsPage({policy}: WithPolicyProps) {
     const styles = useThemeStyles();
     const policyID = policy?.id ?? '';
     const {syncLocations, pendingFields} = policy?.connections?.quickbooksOnline?.config ?? {};
+    const shouldBeDisabled = policy?.connections?.quickbooksOnline?.config.reimbursableExpensesExportDestination !== CONST.QUICKBOOKS_REIMBURSABLE_ACCOUNT_TYPE.JOURNAL_ENTRY;
```

Comment:
Just like we did for `Taxes`, here too we can prevent disable if `Locations` are already imported so that user can toggle off.

Reply:
good point! changed 😄